lib/dartino.dart

```diff
@@ -150,13 +150,13 @@ void _checkSdkInstalled([_]) {
 }
 
 final Duration _checkSdkTimeout = new Duration(seconds: 3);
-bool _skipSdkCheck = true; // skip check on startup
+int _skipSdkCheck = 3; // skip check on startup
 Timer _checkSdkTimer;
 
 /// If an SDK is configured, validate it... but not on startup
 void _checkSdkValid([_]) {
-  if (_skipSdkCheck) {
-    _skipSdkCheck = false;
+  if (_skipSdkCheck > 0) {
```

Comment:
I have this problem as well at startup, where I want to use a value but only perform some action if it changes. There's a commented out method on the atom `Config` class - `onDidChange`. I think you want to set up the sdk initially get getting the value from the config, and then listen for changes via `_checkSdkValid` instead of `observe`.